app/src/main/java/io/zenandroid/onlinego/utils/Globals.kt

```diff
@@ -326,3 +329,9 @@ data class TimerDetails (
     var secondLine: String? = null,
     var timeLeft: Long
 )
+
+fun toastException(t: Throwable, long: Boolean = false) {
```

Comment:
Please don't do this, it makes the viewmodels untestable.

Reply:
It should be testable well enough, you can mock out this call, or make it conditional.

Ok, though, would you be fine with it being `if (DEBUG)` or are you not okay with it on any level?

Reply:
It's goes against all separation principles I'm afraid. Feel free to keep it for your debugging if you need to, but please don't commit it. I assume the logcat in Visual Studio is not at the same level as the one in Android Studio and that's why you'd need this.

Reply:
It's a convenience thing. I use logcat raw, via adb, because I'm not using an IDE, and that makes figuring out the PID of the app each run and restarting logcat with the right filters a pain, better to know there's an error in the first place from a Toast. 

Better for users too, rather than things just failing and them not seeing anything. I guess that's an edge case though, at least in theory...

I'll drop it from the viewmodels but am I fine to leave it in utils?